benefits/core/templates/core/includes/help-links.html

```diff
@@ -0,0 +1,26 @@
+<div class="buttons-agencies">
+    {% for agency_button in buttons %}
+        {% if "agency" in agency_button.classes %}
+            {% if agency_button.label != "phone" %}
+                <strong>{{agency_button.label}}:</strong>
+                {% comment %} <a target="{{ agency_button.target }}"
+                    rel="{{ agency_button.rel }}"
+                    href="{{ agency_button.url }}">
+                    {{ agency_button.label }}:
+                </a> {% endcomment %}
```

Comment:
This code creates a _link_ to the agency, but it's commented out because a link is not part of the Figma designs.

Reply:
Let's try and resolve this question before committing commented code to `dev`... I'd rather not if we can get a solid answer.